src/Gateways/AbstractGateway.php

```diff
@@ -381,6 +402,31 @@ public function init_form_fields() {
 						'maxlength' => 18,
 					),
 				),
+			    'check_avs_cvv' => array(
+			        'title'				=> __( 'Check AVS CVN', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'label'				=> __( 'Check AVS/CVN result codes and reverse transaction.', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'type'				=> 'checkbox',
+			        'description'		=> sprintf(
+			            __( 'This will check AVS/CVN result codes and reverse transaction.' )
+			            ),
+			        'default'			=> 'no'
+			    ),
+			    'avs_reject_conditions'    => array(
+			        'title'       => __( 'AVS Reject Conditions', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'type'        => 'multiselect',
+			        'class'       => 'wc-enhanced-select',
+			        'css'         => 'width: 450px',
+			        'description' => __( 'Choose for which AVS result codes, the transaction must be auto reveresed.'),
+			        'options'     => $this->avs_rejection_conditions(),
+			    ),
+			    'cvn_reject_conditions'    => array(
+			        'title'       => __( 'CVN Reject Conditions', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'type'        => 'multiselect',
+			        'class'       => 'wc-enhanced-select',
+			        'css'         => 'width: 450px',
+			        'description' => __( 'Choose for which CVN result codes, the transaction must be auto reveresed.'),
+			        'options'     => $this->cvn_rejection_conditions(),
```

Comment:
and defaults here too
```suggestion
			        'options'     => $this->cvn_rejection_conditions(),
			        'default'     => array("P", "?", "N"),
```

Reply:
default consitions added